data/tribes/initialization/empire/units.lua

```diff
@@ -1174,6 +1201,10 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_lumberjacks_house",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire production site: Lumberjack's House
+            lore = pgettext("empire_building", "‘Domus lignator.’ (Home of the woodcutter.)"),
```

Comment:
@klaus-halfmann I don't think we should turn these into cases of quidquid latine scriptum est, altum videtur. Surely we can find something better.